Makefile

```
@@ -592,12 +592,16 @@ test-e2e-push-image: $(GINKGO) $(HELM) $(CLUSTERCTL) kubectl e2e-image-push
 	$(E2E_RUN_COMMAND)
 
 .PHONY: e2e-image
-e2e-image: ## Build the image for e2e tests
+e2e-image: ## Build and push the image for e2e tests
```

```suggestion
e2e-image: ## Build the image for e2e tests
```
This is not pushing as far I can understand